### ISSUE TITLE
Can't install on macOS Sierra (10.12.6) — using Erlang 20.3.6

### ISSUE THREAD

**Issue by x80486** [Stratus3D/asdf-rebar#7]
I can't seem to install `rebar3` in my system. I'm always getting these (error) messages:

```shell
[gorre@uplink ~]$ asdf install rebar 3.5.2
** Resuming transfer from byte position 287136
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   120    0   120    0     0    909      0 --:--:-- --:--:-- --:--:--   909
  0  280k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Dependency providers already exists
Dependency getopt already exists
Dependency cf already exists
Dependency erlware_commons already exists
/private/var/folders/w1/cjm29vls6y7fcfdy1wss19rsh5k6h9/T/rebar3-3.5.2/_build/default/lib/erlware_commons/src/ec_dictionary.erl:none: no such file or directory
```

This is some relevant info:

```shell
[gorre@uplink ~]$ uname -a
Darwin uplink 16.7.0 Darwin Kernel Version 16.7.0: Tue Jan 30 11:27:06 PST 2018; root:xnu-3789.73.11~1/RELEASE_X86_64 x86_64

[gorre@uplink ~]$ which erl
/Users/gorre/.asdf/shims/erl

[gorre@uplink ~]$ erl
Erlang/OTP 20 [erts-9.3.1] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:10] [hipe] [kernel-poll:false]

Eshell V9.3.1  (abort with ^G)
1> erlang:system_info(otp_release).
"20"
2> q().
ok
3>

[gorre@uplink ~]$ cat .tool-versions
nodejs 9.9.0
erlang 20.3.6
golang 1.10.2
gradle 4.7
java 8.172
maven 3.5.3
rust stable
```

**Comment by Stratus3D** [comment]
Hmm... I see some compile issues were reported for rebar 3.5.1. Maybe they failed to properly fix everything in 3.5.2? I'd suggest trying 3.5.3 for now.

**Comment by x80486** [comment]
OK, this is not an issue for real. Turns out it's something with the network at my workplace. Who knows! They block everything! I tested it on a co-worker's laptop and it doesn't work. Then I tested it on some friend's (personal) laptop and everything is fine.

Thanks for the tip!
